infra/base-images/base-runner/bad_build_check

```diff
@@ -371,6 +371,22 @@ function check_architecture {
 
 function main {
   local FUZZER=$1
+
+  # Move the directory the fuzzer is located in to somewhere that doesn't exist
+  # on the builder to make it more likely that hardcoding /out fails here (since
+  # it will fail on ClusterFuzz).
+  FUZZER=$(realpath $FUZZER)
+  local INITIAL_FUZZER_DIR=$(dirname $FUZZER)
+
+  local FUZZER_DIR=/tmp/mnt/scratch0/clusterfuzz/bot/builds/build/revisions
```

Comment:
Why such a long dir name, why not just /tmp/build/$(basename $FUZZER)/ ?

Reply:
TO be most similar to ClusterFuzz's directory and less likely to be hardcoded.